nx-X11/lib/X11/XlibInt.c

```diff
@@ -4178,6 +4213,36 @@ Screen *_XScreenOfWindow(Display *dpy, Window w)
     return NULL;
 }
 
+/*
+ * WARNING: This implementation's pre-conditions and post-conditions
+ * must remain compatible with the old macro-based implementations of
+ * GetReq, GetReqExtra, GetResReq, and GetEmptyReq. The portions of the
+ * Display structure affected by those macros are part of libX11's
+ * ABI.
+ */
+void *_XGetRequest(Display *dpy, CARD8 type, size_t len)
```

Comment:
_XGetRequest re-introduced here. Should not have gotten removed in 9c9fe36040a8672608ff8cac2e711d1891f50d47.